modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/data/set/view/table/CPInstancePriceEntryTableFDSView.java

```diff
@@ -39,6 +39,10 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 			"unitPrice", "unit-price"
 		).add(
 			"createDate", "create-date"
+		).add(
+			"priceOnApplication", "price-on-application",
```

Comment:
is there a reason for this sorting? This is the order in which the columns will be shown in the UI, so isn't better to have the createDate at the end (evantually with workflow fields) like other similar tables?

Reply:
done